openghg/standardise/_standardise.py

```diff
@@ -76,6 +76,7 @@ def standardise_surface(
     compressor: Optional[Any] = None,
     filters: Optional[Any] = None,
     chunks: Optional[Dict] = None,
+    optional_metadata: Optional[dict] = None,
```

Comment:
```suggestion
    optional_metadata: Optional[Dict] = None,
```

Reply:
We should be consistent with the use of `Dict` instead of `dict` until we're ready to move over to the direct types like this.